### ISSUE TITLE
Minor doubts

### ISSUE THREAD

**Issue by agamjain2012** [hcmlab/vadnet#23]
1. wanted to check if the URL mentioned in the repo for the training dataset has moved from https://verteiler5.mediathekview.de/Filmliste-akt.xz to https://verteiler1.mediathekview.de/Filmliste-akt.xz

2. I'm training with standard config on this dataset(verteiler1) and my accuracy after 5 epochs is still going between 0.6 to 0.7. is this the typical accuracy we should expect on the dataset?

any help/recommendation/suggestion is welcome
